packages/web-components/src/components/va-privacy-agreement/test/va-privacy-agreement.e2e.ts

```diff
@@ -151,4 +151,153 @@ describe('va-privacy-agreement', () => {
+  });
+
+  it('uswds v3 passes an aXe check - no error', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(
+      '<va-privacy-agreement checked="true" uswds/>',
+    );
+
+    await axeCheck(page);
+  });
+
+  it('uswds v3 checkbox should be checked if the `checked` prop is present', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(
+      '<va-privacy-agreement checked uswds/>',
+    );
+
+    const vaCheckbox = await page.find('va-privacy-agreement >>> va-checkbox');
+    const checkedValue = await vaCheckbox.getProperty("checked");
+
+    expect(checkedValue).toBe(true);
+  });
+
+  it('uswds v3 checkbox should not be checked if the `checked` prop is not present', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(
+      '<va-privacy-agreement uswds/>',
+    );
+
+    const vaCheckbox = await page.find('va-privacy-agreement >>> va-checkbox');
+    const checkedValue = await vaCheckbox.getProperty("checked");
+
+    expect(checkedValue).toBeFalsy();
+  });
+
+  /** 
+   * Skipping the following tests because accessing the shadowRoot of the checkbox
+   * in order to trigger an input click is flakey. The tests pass locally but fail in CI.
+   */
+  it.skip('uswds v3 emits the vaChange event', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<va-privacy-agreement uswds/>',
+    );
+    const changeSpy = await page.spyOnEvent('vaChange');
+    const input = (
+      await page.waitForFunction(() =>
+        document.querySelector('va-privacy-agreement')
+          .shadowRoot.querySelector('va-checkbox')
+          .shadowRoot.querySelector('input')
+      )
+    ).asElement();
+
+    await input.click();
+
+    expect(changeSpy).toHaveReceivedEventDetail({ checked: true });
+  });
+
+  it.skip('uswds v3 fires analytics event when enableAnalytics prop is set', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<va-privacy-agreement enable-analytics uswds/>',
+    );
+    const analyticsSpy = await page.spyOnEvent('component-library-analytics');
+    const input = (
+      await page.waitForFunction(() =>
+        document.querySelector('va-privacy-agreement')
+          .shadowRoot.querySelector('va-checkbox')
+          .shadowRoot.querySelector('input')
+      )
+    ).asElement();
+
+    input.click();
+    await page.waitForChanges();
+
+    expect(analyticsSpy).toHaveReceivedEventDetail({
+      action: 'click',
+      componentName: 'va-privacy-agreement',
+      details: {
+        checked: true
+      },
+    });
+
+    input.click();
+    await page.waitForChanges();
+
+    expect(analyticsSpy).toHaveReceivedEventDetail({
+      action: 'click',
+      componentName: 'va-privacy-agreement',
+      details: {
+        checked: false
+      },
+    });
+
+  });
+
+  it.skip('uswds v3 does not fire analytics event when `enableAnalytics` prop is not set', async () => {
```

Comment:
There are a couple of tests added that are skipped?

Reply:
A couple of tests, like V1, were initially skipped. However, upon re-evaluation, they appear to be passing now, so I have removed the skip.